packages/paste-core/components/toast/src/Toaster.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import {Stack} from '@twilio-paste/stack';
+
+export interface ToasterProps {
+  children: NonNullable<React.ReactNode>;
+}
+
+const Toaster: React.FC<ToasterProps> = ({children, ...props}) => {
+  return (
+    <Box position="fixed" right="space40" top="space40" zIndex="zIndex90" {...safelySpreadBoxProps(props)}>
```

Comment:
@SiTaggart should we add a placement prop now that controls whether the toaster is placed? Or wait until we further flush out the toaster?

Reply:
So firstly lets not call this a toaster. This can just be the toast container and it only cares about positioning the toasts on the screen relative to the window.

On the placement prop, I'm not sure we're even allowing you to change it. It should be fixed for every page and every app so toasts always appear in the same position.

Reply:
Renamed Toaster to ToastContainer.